docker/buildDocker.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+################################################################################
+# Build docker locally
+# Usage:
+# bash buildDocker.sh [TAG_NAME]
+################################################################################
+
+################################################################################
+# Define default values for variables
+################################################################################
+TAG_NAME=local
+
+################################################################################
+# START DECLARATION FUNCTIONS
+################################################################################
+
+################################################################################
+function usage {
+################################################################################
+  echo USAGE:
+  echo   $0 [TAG_NAME]
+  exit 1
+}
+
+################################################################################
+function printInfo {
+################################################################################
+echo ---------------------------------------------------------------------------
+echo $*
+echo ---------------------------------------------------------------------------
+}
+
+################################################################################
+# END DECLARATION FUNCTIONS / START OF SCRIPT
+################################################################################
+
+################################################################################
+# Test for commands used in this script
+################################################################################
+testForCommands="dirname date docker git"
+
+for command in $testForCommands
+do 
+  $command --help >> /dev/null
```

Comment:
Not sure about linux, but on OSX (10.15.7) "--help" is not available for `dirname` and `date`. I think `type $command` could work more cross platform? Not sure what the most accepted solution is.

```suggestion
  type $command >> /dev/null
```